components/automate-ui/src/app/page-components/data-feed-table/data-feed-table.component.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  Component,
+  OnInit,
+  OnChanges,
+  Input } from '@angular/core';
+import { MatOptionSelectionChange } from '@angular/material/core';
+import { DeleteDestination, EnableDisableDestination } from 'app/entities/destinations/destination.actions';
+import { Destination } from 'app/entities/destinations/destination.model';
+import { Observable, Subject } from 'rxjs';
+import { select, Store } from '@ngrx/store';
+import { NgrxStateAtom } from 'app/ngrx.reducers';
+import { filter, takeUntil } from 'rxjs/operators';
+import { EntityStatus, pending } from 'app/entities/entities';
+import { destinationEnableStatus } from 'app/entities/destinations/destination.selectors';
+
+@Component({
+  selector: 'app-data-feed-table',
+  templateUrl: './data-feed-table.component.html',
+  styleUrls: ['./data-feed-table.component.scss']
+})
+export class DataFeedTableComponent implements OnInit, OnChanges {
```

Comment:
```suggestion
export class DataFeedTableComponent {
```
We should remove OnInit and OnChanges if we are not implementing them

Reply:
Done